python/sbp/client/examples/bytes.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Mark Fine <mark@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+from sbp.client.drivers.pyserial_driver import PySerialDriver
+from sbp.client.loggers.byte_logger     import ByteLogger
+from sbp.client.handler                 import Handler
+
+DEFAULT_SERIAL_PORT  = "/dev/ttyUSB0"
+DEFAULT_SERIAL_BAUD  = 1000000
+DEFAULT_LOG_FILENAME = time.strftime("sbp-%Y%m%d-%H%M%S.log")
+
+
+def get_args():
+  """
+  Get and parse arguments.
+  """
+  import argparse
+  parser = argparse.ArgumentParser(description="Swift Navigation SBP Bytes Example.")
```

Comment:
Maybe clarify what the program does a little more: "Parses an SBP stream from serial and writes it to a file in the same binary format."